From e40f7e425aca3a5718df935752ef296640bd3c9a Mon Sep 17 00:00:00 2001
From: LeeRowe1 <lee.rowe@bjss.com>
Date: Wed, 19 Oct 2022 10:19:17 +0100
Subject: [PATCH 2/2] Add newer dev work

---
 newFile2 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/newFile2 b/newFile2
index d84e45a..23ad327 100644
--- a/newFile2
+++ b/newFile2
@@ -1,2 +1,2 @@
 Another new file
-and a new awesome feature
\ No newline at end of file
+and a new awesome featurenewer dev work
-- 
2.37.3.windows.1

